receiver/sapnetweaverreceiver/README.md

```diff
@@ -0,0 +1,54 @@
+# SAP Netweaver Receiver
+This receiver collects metrics from SAP Netweaver instance based on the [SAPControl Web Service Interface](https://www.sap.com/documents/2016/09/0a40e60d-8b7c-0010-82c7-eda71af511fa.html). Stats are collected from the `GetAlertTree` and `EnqGetLockTable` methods.
+
+## Supported Pipelines
+- Metrics
+
+## How It Works
+1. The user configures this receiver in a pipeline.
+2. The user configures a supported component to route telemetry from this receiver.
+
+## Prerequisites
+- SAP Netweaver 7.10+
+- The ability to authenticate to the OS
+- SAP read-only permission
+- For SAP version 7.38+,  (Windows: execute permission, Unix: write permission) is required for each instance of sapstartsrv on the host. If this permission is being assigned to a group, the - monitoring user in the group must have the group set as primary. If authentication or authorization check fails the request will fail with “Invalid Credentials" or "Permission denied" fault string.
+
+More information on how to setup a SAP NetWeaver Stack for each operating system and version can be found [here](https://help.sap.com/docs/SAP_NETWEAVER/9e41ead9f54e44c1ae1a1094b0f80712/576f5c1808de4d1abecbd6e503c9ba42.html?language=en-US).
+
+## Configuration
+| Field               | Type               | Default                                                                                   | Description                                                                                                                                                  |
+|---------------------|--------------------|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| metrics             | map             | (default: see `DefaultMetricsSettings` [here](./internal/metadata/generated_metrics.go) | Allows enabling and disabling specific metrics from being collected in this receiver.                                                                        |
```

Comment:
```suggestion
| metrics             | map             | (default: see `DefaultMetricsSettings` [here](./internal/metadata/generated_metrics.go) | Allows enabling and disabling [specific metrics](./documentation.md#Metrics) from being collected in this receiver.                                                                        |
```

May be nice to point to better documentation for doing this

Reply:
updated